config/nxf_vcf.config

```diff
@@ -127,6 +127,7 @@ params {
       max_records = ""
       max_samples = ""
       template = ""
+      include_crams = true
```

Comment:
- add parameter validation
- consistency: move some lines up to maintain alphabetic ordering